Log outgoing messages, make `logMessage` func

sethetter opened the pull request:
Closes #8

@jcbwlkr did I do it right?!

sethetter committed:
Log outgoing messages, make `logMessage` func

Closes #8

jcbwlkr commented on main.go:
```diff
@@ -34,10 +34,7 @@ func twilioIncomingHandler(rw http.ResponseWriter, r *http.Request) {
-		respondError(rw, err)
-		return
-	}
+	logMessage(rw, from, to, body)
```
If the db call fails inside `logMessage` we will set the status code for the response but we don't otherwise abort the handler. Are we ok with that?

sethetter replied:
Probably not. Would the thing to do be returning the error from the `db.Exec` command?

jcbwlkr replied:
yeah probably. Let the handler decide what to do if it fails.

jcbwlkr replied:
Also means the `logMessage` func doesn't need the response writer anymore

jcbwlkr commented:
All three of these cases do the same thing but just with a different value for `message`. What if we do like

```
var message string

switch body {
case "signup":
  message = "a"
case "stop":
  message = "b"
default:
  message = "c"
}
logMessage(rw, to, from, message)
rw.Write([]byte(message))
```

sethetter committed:
Handle `logMessage` error in handler body

Also move to single `logMessage` call, with message set in the switch statement.

Progress #8

sethetter commented:
@jcbwlkr Updated! Care to give it a :+1: or a :-1: ?

jcbwlkr commented:
:+1:
